netbox/tenancy/forms/filtersets.py

```diff
@@ -71,7 +71,7 @@ class ContactRoleFilterForm(NetBoxModelFilterSetForm):
 
 class ContactFilterForm(NetBoxModelFilterSetForm):
     model = Contact
-    group_id = DynamicModelMultipleChoiceField(
+    contact_group_id = DynamicModelMultipleChoiceField(
```

Comment:
If we keep the filter name
```suggestion
    group_id = DynamicModelMultipleChoiceField(
```